src/main/java/Event.java

```diff
@@ -0,0 +1,19 @@
+class Event extends Task {
+    public Event(String message) {
+        super(message);
+    }
+
+    @Override
+    public String provideDetails() {
```

Comment:
I feel that `provideDetails()` is not that intuitive to users, why did you choose not to `@Override` the default `toString()` method in Object class? I think that will be a better design, feel free to correct me.

Reply:
That is a very good suggestion. Thanks for suggesting, I will implement `toString()` instead. Thanks @yyj-02.